schema.graphql

```diff
@@ -71,14 +71,28 @@ type Dispute @entity {
   lastRoundId: BigInt!
   state: DisputeState!
   settledPenalties: Boolean!
-  metadata: String!
+  metadata: Bytes!
+  disputable: Disputable @derivedFrom(field: "dispute")
   rounds: [AdjudicationRound!] @derivedFrom(field: "dispute")
   jurors: [JurorDispute!] @derivedFrom(field: "dispute")
   txHash: String!
   createdAt: BigInt!
   ruledAt: BigInt
 }
 
+type Disputable @entity {
```

Comment:
What would this look like for the existing (and non-Agreement) disputes? Just non-existent?

Reply:
Yes, do you think it would be better doing it differently?

Reply:
No, I think it makes sense. With all the other disputes, I guess we can't assume anything from their metadata?

My worry is if we'll need a more abstract concept when we need to consider more types of disputes.

Reply:
Exactly, I think for now we are safe to consider only these two types: raw disputes and agreements' disputes